crates/polars-arrow/src/legacy/kernels/rolling/no_nulls/min_max.rs

```diff
@@ -286,6 +287,112 @@ macro_rules! rolling_minmax_func {
 rolling_minmax_func!(rolling_min, MinWindow, compute_min_weights);
 rolling_minmax_func!(rolling_max, MaxWindow, compute_max_weights);
 
+macro_rules! minmax_window_bool {
```

Comment:
I only wnat to use macro's for dispatch. The actual functions should be implemented with generics. Can you rewrite this to use generics?